pram

```diff
@@ -149,6 +156,24 @@ main() {
 			-S|--no-signoff)
 				signoff=
 				;;
+			-p|--part-of)
+				partof=1
+				;;
+			--part-of-pr)
+				partof=1
+				[[ -z ${user_pr} ]] || die "${0}: cannot specify multiple ${1}"
+				[[ ${#} -gt 1 ]] || die "${0}: missing argument to ${1}"
+				user_pr=${2}
+				shift
+				;;
+			--part-of-pr=*)
```

Comment:
In general, I think it's a good idea. But I think it should also override what we put as `Closes:` implicitly, i.e. we are effectively having a series of `Part-of` trailers, followed by a `Closes` trailer that should always be referencing the same bug.

As such, we probably need a better name too.